app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -1501,7 +1500,7 @@ BrowserWorld::UpdateVisibleWidgets() {
   });
 
   for (const WidgetPtr& widget: widgets) {
-    if (widget->IsVisible() && !widget->IsResizing()) {
+    if (widget->IsVisible()) {
```

Comment:
I guess you forgot to restore this condition?

Reply:
No, it's actually needed, see the PR description.